.github/workflows/scheduled_scan.yaml

```diff
@@ -0,0 +1,35 @@
+name: Scheduled scan
```

Comment:
We could run the main workflow itself on a schedule instead of having a separate scheduled scan.
What do you think?

Reply:
This workflow is meant to scan the published rock. If we run the main workflow, we will rebuild the image and scan that new image instead. It might make sense to also do that on a schedule, but I think we will need both in the long run. When we have multiple version of the same image published, it would be more efficient to just scan them instead of rebuilding all of them.